themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -30,7 +30,8 @@
     {if $product.show_quantities}
       <div class="product-quantities">
         <label class="label">{l s='In stock' d='Shop.Theme.Catalog'}</label>
-        <span>{$product.quantity} {$product.quantity_label}</span>
+        <span data-stock="{$product.quantity}">{$product.quantity} {$product.quantity_label}</span>
+        <input type="hidden" id="allow_oosp" value="{$product.allow_oosp}">
```

Comment:
I agree, I'm using it to store information for the javascript code.
I've replaced it by data attribute.

Thanks !